From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-shmem-support-vm_uffd_ops-api-checkpatch-fixes
Date: Fri Jun 27 02:25:37 PM PDT 2025

WARNING: please, no space before tabs
#56: FILE: mm/shmem.c:5216:
+^I.uffd_features^I= ^I__VM_UFFD_FLAGS,$

WARNING: please, no space before tabs
#57: FILE: mm/shmem.c:5217:
+^I.uffd_ioctls^I= ^IBIT(_UFFDIO_COPY) |$

Cc: Peter Xu <peterx@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/shmem.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/mm/shmem.c~mm-shmem-support-vm_uffd_ops-api-checkpatch-fixes
+++ a/mm/shmem.c
@@ -5213,8 +5213,8 @@ static int shmem_error_remove_folio(stru
 
 #ifdef CONFIG_USERFAULTFD
 static const vm_uffd_ops shmem_uffd_ops = {
-	.uffd_features	= 	__VM_UFFD_FLAGS,
-	.uffd_ioctls	= 	BIT(_UFFDIO_COPY) |
+	.uffd_features	=	__VM_UFFD_FLAGS,
+	.uffd_ioctls	=	BIT(_UFFDIO_COPY) |
 				BIT(_UFFDIO_ZEROPAGE) |
 				BIT(_UFFDIO_WRITEPROTECT) |
 				BIT(_UFFDIO_CONTINUE) |
_
